index.js

```diff
@@ -1,63 +1,64 @@
+#!/usr/bin/env node
+const prog = require('caporal');
 var Promise = require("bluebird");
 var input = require('./input');
 var output = require('./output');
 
-if(process.argv.length<4){
-	console.log("Usage: node index,js <path_to_team.csv_file> <path_to_scores.csv_file>\n\nOutputs a teamScores.csv file to the current directory");
-	process.exit(1);
-}
+prog
+    .version('1.0.0')
+    .description('Program to facilitate pairwise evaluation in CS F213 Labs')
+    .command('pair_eval', 'Compute pairwise evaluation scores.')
+    .argument('<teamcsv>', 'CSV Data for Teams')
+    .argument('<scorescsv>', 'CSV Data for Scores')
+    .action(function(args, options, logger) {
+        if(Object.keys(args).length != 2){
```

Comment:
Can you expose this functionality via a different file, say `main.js` and then separate caporal and the functionality itself.